src/DistributedTracingManager.ts

```diff
@@ -152,45 +156,33 @@ export class DistributedTracingManager extends BaseExplorer {
         }
 
         if (deviceIds.length === 1) {
-            try {
-                let registry = iothub.Registry.fromConnectionString(iotHubConnectionString);
-                await  registry.updateTwin(deviceIds[0], JSON.stringify(twinPatch), twinPatch.etag);
-            } catch (err) {
-                return err.message;
-            }
+            let registry = iothub.Registry.fromConnectionString(iotHubConnectionString);
+            await registry.updateTwin(deviceIds[0], JSON.stringify(twinPatch), twinPatch.etag);
             return;
         }
 
-        const result = this.scheduleTwinUpdate(twinPatch, iotHubConnectionString, deviceIds);
-        return result;
+        return await this.scheduleTwinUpdate(twinPatch, iotHubConnectionString, deviceIds);
```

Comment:
No need for `return await`, just `return`

Reply:
fixed